docs/deployment/landing_page.rst

```diff
@@ -258,11 +258,11 @@ that includes a file integrity monitor. More information can be found
 **Don't log access to the *Landing Page* in the webserver**
 
 Here's an Apache example that would exclude the *Landing Page* from
-logging:
+logging. However you still need to make sure no other assets get logged!
 
 ::
 
-    SetEnvIf Request_URI "^/securedrop$" dontlog
+    SetEnvIf Request_URI "^/securedrop(/.*)$" dontlog
```

Comment:
This wouldn't catch the URI `/securedrop` which the original does. You could change that to `^/securedrop($|/)` which would be a minimal rule that catches this (assuming Apache behaves like `grep`).

Reply:
Good catch, I had a rewrite that would hit before so you would always hit example.com/securedrop/ (note that last slash). I'll assign to myself so I won't forget to triple check something better as your suggestion (it wouldn't catch it either).